UndertaleModCli/Program.cs

```diff
@@ -341,8 +341,14 @@ private static int Dump(DumpOptions options)
             return EXIT_FAILURE;
         }
 
+        if (program.Data.IsYYC())
+        {
+            Console.WriteLine("Game is YYC, there is no code to dump. Exiting.");
```

Comment:
I would suggest something like:
```suggestion
            Console.WriteLine("The game uses YYC (YoYo Compiler), which means the code is embedded into the game executable, so there is no code to dump. Exiting.");
```

Though I'm not sure if it's necessary

Reply:
Too used from coreutils, where we have short sentences. I'll rephrase it a little.